lib/src/main/java/com/auth0/android/lock/PasswordlessLockActivity.java

```diff
@@ -400,14 +404,14 @@ public void onCountryCodeChangeRequest(CountryCodeChangeEvent event) {
     @SuppressWarnings("unused")
     @Subscribe
     public void onPasswordlessAuthenticationRequest(PasswordlessLoginEvent event) {
-        if (configuration.getDefaultPasswordlessStrategy() == null) {
+        if (configuration.getPasswordlessConnection() == null) {
             Log.w(TAG, "There is no default Passwordless strategy to authenticate with");
             return;
         }
 
         lockView.showProgress(true);
         AuthenticationAPIClient apiClient = new AuthenticationAPIClient(options.getAccount());
-        String connectionName = configuration.getFirstConnectionOfStrategy(configuration.getDefaultPasswordlessStrategy());
+        String connectionName = configuration.getPasswordlessConnection().getName();
```

Comment:
@hzalaz That's handled before showing the widget, it will show an error if no connections are available for the "Passwordless mode" (needs social || passwordless) -> https://github.com/auth0/Lock.Android/blob/1d31ce2fcd3a46cea6728070f3158317f2976837/lib/src/main/java/com/auth0/android/lock/views/PasswordlessLockView.java#L115
Also, in case the event is received but again no passwordless connection is available, it will break before calling authenticate -> https://github.com/auth0/Lock.Android/pull/335/files#diff-396f039685bb2421bd638efb3baadd3eR407